docs/src/main/sphinx/release/release-431.md

```diff
@@ -0,0 +1,55 @@
+# Release 431 (25 Oct 2023)
+
+## General
+
+* Add support for the `CREATE OR REPLACE TABLE` statement. ({issue}`13180`)
```

Comment:
`CREATE TABLE` already exists. Technically, this adds `REPLACE TABLE`. It can only be used in combination with `CREATE TABLE`, in the form of `CREATE OR REPLACE TABLE`.

Reply:
I know .. how do we say that ... 

Add support for `OR REPLACE` with `CREATE TABLE` statements.  

??